modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ResultsBar.js

```diff
@@ -31,10 +31,11 @@ const ResultsBar = ({
 					<span className="component-text text-truncate-inline">
 						<span className="text-truncate">
 							{Liferay.Util.sub(
-								Liferay.Language.get('x-results-for-x'),
-								itemsTotal,
-								searchValue || ''
+								Liferay.Language.get('x-results-for'),
+								itemsTotal
 							)}
+
+							<strong>{` "${searchValue}"`}</strong>
```

Comment:
I believe that we do not need the `"` around the `${searchValue}` inside the template literal here. [Here's the docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). I might be wrong, so do double-check please ❤️

Reply:
Hey @kresimir-coko! 😄 . Yeah, that is what I've done first but the keyword result must be between `"`, 
![bold_results](https://user-images.githubusercontent.com/44723449/158141041-a756f88d-44ed-4ed3-bdff-cb3e45432e92.png)


that's why I put them. But I will check it too ❤️ .

Reply:
Ah yes, in that case you need them! Well done!

Reply:
To simplify that could be

```
<strong>"{searchValue}"<strong>
```